drupal-migrate.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# This script will help to migrate data from a Drupal bot.
+# https://www.drupal.org/project/bot
+#
+# In order to do this we need to have access to the MySQL database for the
+# Drupal bot, and the lubot mongodb database. You also need to have either the
+# lubot started, or at least the mongodb instance needs to be running.
+
+# Help text.
+usage() { echo "Usage: $0 [-m arguments to pass to MySQL] [-d name of the mongodb database that contains the bots brain] -c [IRC channel name to assign imported data to]" 1>&2; exit 1; }
+
+# Parse arguments from command line.
+while getopts ":m:d:c:" o; do
+    case "${o}" in
+        m)
+            MYSQL_STRING=${OPTARG}
+            ;;
+        d)
+            MONGO_STRING=${OPTARG}
+            ;;
+        c)
+            CHANNEL=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+# Verify required arguments are present.
+if [ -z "${MYSQL_STRING}" ] || [ -z "${MONGO_STRING}" ]; then
+    usage
+fi
+
+# Where are we executing this script?
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+# Cleanup from any past runs that failed.
+rm $DIR/karma.csv;
+rm $DIR/factoids.csv;
+rm $DIR/logs.csv;
```

Comment:
These commands are going to fail if the files don't exist. You might want to pass -f.